src/pages/resultsView/enrichments/styles.module.scss

```diff
@@ -28,6 +28,10 @@
     font-size: 11px;
 }
 
+.TableContainer input {
```

Comment:
i think this might be to general because there are other inputs inside the container no?